lib/ardb/adapter/base.rb

```diff
@@ -1,16 +1,21 @@
+require 'ardb'
+
 module Ardb; end
-class Ardb::Adapter
+module Ardb::Adapter
 
   class Base
 
-    attr_reader :ar_connect_hash, :database
+    attr_reader :config, :ar_connect_hash, :database
```

Comment:
@jcredding also applies to the `public_ar_connect_hash` method on the postgres adapter.